### ISSUE TITLE
Add Example Apps

### ISSUE THREAD

**Issue by iamprakash13** [iamprakash13/Flutter-Widgets#3]
Add some examples at last of Readme.md with new title 'examples'. that include demo with  link to proper site/ code.

happy hacking!

**Comment by AlKun25** [comment]
Could you elaborate?
Cause I think I can work on this issue

**Comment by iamprakash13** [comment]
yep, it just adding apps made with flutter as demo/clone/example.

you can refer to other issues as well. either you can add default widgets usage or custom widget that you made.

**Comment by hemangdtu** [comment]
I am practicing Flutter since 5-6 months and I am very fascinated towards Flutter. I wish to make my first Flutter application which was a Calculator App a part of my first HacktoberFest. If it is fine with you then I request you to assign this task to me. Thank you.

**Comment by iamprakash13** [comment]
yes @hemangdtu go ahead. make your application add to example apps section in this repo.

**Comment by mayankparida** [comment]
Could I add an app I made called autozoom which allows you to join zoom calls automatically at the desired time? It is really easy to make and would be great for your learning resource. Thank you!

**Comment by iamprakash13** [comment]
sure @mayankparida
